live_pipeline/requirements.txt

```diff
@@ -0,0 +1,8 @@
+boto3==1.28.36
+botocore==1.31.36
+jmespath==1.0.1
+python-dateutil==2.8.2
+python-dotenv==1.0.0
+s3transfer==0.6.2
+six==1.16.0
+urllib3==1.26.16
```

Comment:
Since this will later be added to a Docker image, we only want to include the libraries that have been specifically installed as the image will end up much larger otherwise. You probably only want to include `boto3`, `python-dotenv`, `python-dotenv`